### PR TITLE
Corrected OpCodes for Battlefield / Arena

### PR DESCRIPTION
**Pull request by iCry1337**
Renamed Wrong Opcode
Added TolBarad in SwitchCase to BF Command

**Changes proposed**:

- Corrected Opcodes
- Renamed Opcode

**Issues addressed**: Fixes #

**Tests performed**: (Does it build, tested in-game, etc)
- [X] Build Successfull
- [X] Tested Ingame

**Known issues and TODO list**:

- Sometimes it may not popup the invite window
- Maybe it can crash?
- Need more PR tests!

- [ ] CMSG_BATTLEFIELD_MGR_ADVANCE_STATE
- [ ] CMSG_BATTLEFIELD_MGR_SET_NEXT_TRANSITION_TIME
- [ ] SMSG_BATTLEFIELD_MGR_ENTERING

### CODE REVIEW

File: src/server/game/Server/Protocol/Opcodes.h [hunk 1]
```diff
@@ -646,8 +648,9 @@ enum OpcodeServer
     SMSG_BATTLEFIELD_STATUS_NEEDCONFIRMATION          = 0x59A0,
     SMSG_BATTLEFIELD_STATUS_WAITFORGROUPS             = 0x75A2,
     SMSG_BATTLEFIELD_STATUS_FAILED                    = 0x71A7,
-    SMSG_BATTLEGROUND_INFO_THROTTLED                  = 0x34B2,
+    SMSG_BATTLEGROUND_INFO_THROTTLED                  = 0x14A7, //old 0x34B2,
```
Comment (acidmanifesto) [inline src/server/game/Server/Protocol/Opcodes.h:651]: Are you certain this is for cata? Looks very simular to a MOP client. How was this obtained along with the others?

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

File: src/server/game/Server/Protocol/Opcodes.h (hunk 1 above)
Comment (iCry1337) [reply src/server/game/Server/Protocol/Opcodes.h:651]: i had a look into mangos forums on there packet database that was updated 2019 for each client and i checked without any issues. correct me if i am wrong 

https://www.getmangos.eu/wiki/referenceinfo/packetinfo/mangosthreepacket/SMSG_BATTLEGROUND_INFO_THROTTLED-r13303/

MoP 
https://www.getmangos.eu/wiki/referenceinfo/packetinfo/mangosfourpacket/SMSG_BATTLEGROUND_INFO_THROTTLED-r490/

Mop has 0x1E1E

**Comment by Ovahlord** [comment]
rejecting this as 'read this on a forum' is not a reliable information source. If you are touching packets or opcodes, reverse engineer them.

**Comment by funjoker** [comment]
Just checked like 50% of the changed opcodes.
They are literally all wrong.

Keep the TC ones

**Pull request closed without merging**